workspaces/estree-ast-utils/test/utils.ts

```diff
@@ -17,10 +17,14 @@ export function codeToAst(code) {
   return estreeRootNode.body;
 }
 
-export function getExpressionFromStatement(node) {
+export function getExpressionFromStatement(node: any) {
```

Comment:
As `node` could be complex to type I wonder if we could not create a Node type which points to `any` until we found better.